tests/headingscommand.js

```diff
@@ -53,6 +53,15 @@ describe( 'HeadingsCommand', () => {
 				expect( command.value ).to.equal( format );
 			} );
 		}
+
+		it( 'should be equal to defaultFormat if format has not been found', () => {
```

Comment:
OK, so I'm fine with defaulting to the default format in such a case. But next time please do not squeeze such changes without mentioning them ;)